source/local/local.go

```diff
@@ -143,9 +156,18 @@ func (s *localSource) Discover() error {
 			}
 			featuresFromFiles[k] = v
 		}
+
+		// Merge labels from hooks and files
+		for k, v := range labelsFromHooks {
+			if old, ok := labelsFromFiles[k]; ok {
+				klog.InfoS("overriding label value", "labelKey", k, "oldValue", old, "newValue", v)
```

Comment:
The old/existing log message above should be reworded to `overriding feature value..."

Reply:
Yes, one from line 154.